app/models/achievement.rb

```diff
@@ -5,4 +5,9 @@ class Achievement < ActiveRecord::Base
   validates :title, :side, :points, presence: true
 
   enum side: %w(corp runner)
+
+  def to_s
+    "#{self.class.name} [#{self.side}/#{self.points}] '#{self.title}' / '#{self.description}'"
+  end
+
```

Comment:
Well... _damn._ That's a style thing I'd not noticed. I'll clean that up shortly.